pkgs/applications/networking/instant-messengers/telegram/tdesktop/tg_owt.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub, cmake, ninja, yasm
+, pkg-config, libjpeg, openssl, libopus, ffmpeg, alsaLib, libpulseaudio
+}:
+
+let
+  rev = "c73a4718cbff7048373a63db32068482e5fd11ef";
+  sha256 = "0nr20mvvmmg8ii8f2rljd7iv2szplcfjn40rpy6llkmf705mwr1k";
+
+in stdenv.mkDerivation {
+  pname = "tg_owt";
+  version = "git-${rev}";
```

Comment:
FYI: Normally https://nixos.org/manual/nixpkgs/stable/#sec-package-naming applies:
> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".

But in this case it shouldn't really matter since `tg_owt` is only an "internal" package for `tdesktop`.